basic/dapps/ethereum-provider/src/App.tsx

```diff
@@ -0,0 +1,68 @@
+import "./App.css";
+
+import { EthereumProvider } from "@walletconnect/ethereum-provider";
+import { ethers } from "ethers";
+import React, { useState } from "react";
+
+const projectId = import.meta.env.VITE_PROJECT_ID as string;
+
+// 1. Create a new EthereumProvider instance
+const provider = await EthereumProvider.init({
+  projectId,
+  chains: [1],
+  methods: ["eth_signTypedData", "eth_signTypedData_v4"],

```

Comment:
```suggestion
  methods: ["personal_sign", "eth_sendTransaction"],
```
Given this is a basic example, should we use (especially as required) methods that are supported by every wallet? Imho, we should avoid any friction when devs attempt to use & connect using these examples